filebeat/crawler/prospector_log.go

```diff
@@ -140,6 +140,9 @@ func (p *ProspectorLog) scanGlob(glob string) {
 		// Track the stat data for this file for later comparison to check for
 		// rotation/etc
 		p.prospectorList[h.Path] = *h.Stat
+		// TODO: Registrar now updated also if no changes happened. Improve efficiency here
+		// Update registrar
+		p.Prospector.registrar.Persist <- h.GetState()
```

Comment:
what if we do not start an harvester for said file (e.g. due to ignore_older)?

As we require an active harvester here, maybe have harvester send initial event to registrar with offset 0?